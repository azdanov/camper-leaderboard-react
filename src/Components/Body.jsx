// @flow

import React from 'react';

const Body = () => (
  <div className="container">
    <h2>Hello2</h2>
  </div>
);

export default Body;
