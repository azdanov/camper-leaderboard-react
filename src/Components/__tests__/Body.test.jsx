// @flow

import React from 'react';

import { shallow } from 'enzyme';
import Body from '../Body';

test('Head renders correctly', () => {
  const component = shallow(<Body />);
  expect(component).toMatchSnapshot();
});
