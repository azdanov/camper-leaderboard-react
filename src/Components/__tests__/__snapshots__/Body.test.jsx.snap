// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Table
    as="table"
    celled={true}
    fixed={true}
    selectable={true}
    sortable={true}
    striped={true}
    textAlign="center"
  >
    <TableHeader
      as="thead"
    >
      <TableRow
        as="tr"
        cellAs="td"
      >
        <TableHeaderCell
          as="th"
          onClick={[Function]}
          sorted={null}
        >
          Camper
        </TableHeaderCell>
        <TableHeaderCell
          as="th"
          onClick={[Function]}
          sorted={null}
        >
          Points in past 30 days
        </TableHeaderCell>
        <TableHeaderCell
          as="th"
          onClick={[Function]}
          sorted={null}
        >
          All time points
        </TableHeaderCell>
        <TableHeaderCell
          as="th"
          onClick={[Function]}
          sorted={null}
        >
          Last Active
        </TableHeaderCell>
      </TableRow>
    </TableHeader>
    <TableBody
      as="tbody"
    />
    <TableFooter
      as="tfoot"
      fullWidth={true}
    >
      <TableRow
        as="tr"
        cellAs="td"
      >
        <TableHeaderCell
          as="th"
          colSpan="4"
        >
          Built with:
          <List
            bulleted={true}
            horizontal={true}
          >
            <ListItem
              as="a"
              href="https://facebook.github.io/react/"
            >
              React
            </ListItem>
            <ListItem
              as="a"
              href="https://react.semantic-ui.com/"
            >
              Semantic UI React
            </ListItem>
          </List>
        </TableHeaderCell>
      </TableRow>
    </TableFooter>
  </Table>,
  "nodes": Array [
    <Table
      as="table"
      celled={true}
      fixed={true}
      selectable={true}
      sortable={true}
      striped={true}
      textAlign="center"
    >
      <TableHeader
        as="thead"
      >
        <TableRow
          as="tr"
          cellAs="td"
        >
          <TableHeaderCell
            as="th"
            onClick={[Function]}
            sorted={null}
          >
            Camper
          </TableHeaderCell>
          <TableHeaderCell
            as="th"
            onClick={[Function]}
            sorted={null}
          >
            Points in past 30 days
          </TableHeaderCell>
          <TableHeaderCell
            as="th"
            onClick={[Function]}
            sorted={null}
          >
            All time points
          </TableHeaderCell>
          <TableHeaderCell
            as="th"
            onClick={[Function]}
            sorted={null}
          >
            Last Active
          </TableHeaderCell>
        </TableRow>
      </TableHeader>
      <TableBody
        as="tbody"
      />
      <TableFooter
        as="tfoot"
        fullWidth={true}
      >
        <TableRow
          as="tr"
          cellAs="td"
        >
          <TableHeaderCell
            as="th"
            colSpan="4"
          >
            Built with:
            <List
              bulleted={true}
              horizontal={true}
            >
              <ListItem
                as="a"
                href="https://facebook.github.io/react/"
              >
                React
              </ListItem>
              <ListItem
                as="a"
                href="https://react.semantic-ui.com/"
              >
                Semantic UI React
              </ListItem>
            </List>
          </TableHeaderCell>
        </TableRow>
      </TableFooter>
    </Table>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Body />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Body {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "fetchData": [Function],
        "handleSort": [Function],
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "column": null,
          "data": Array [],
          "direction": null,
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Table
          as="table"
          celled={true}
          fixed={true}
          selectable={true}
          sortable={true}
          striped={true}
          textAlign="center"
        >
          <TableHeader
            as="thead"
          >
            <TableRow
              as="tr"
              cellAs="td"
            >
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                Camper
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                Points in past 30 days
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                All time points
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                Last Active
              </TableHeaderCell>
            </TableRow>
          </TableHeader>
          <TableBody
            as="tbody"
          />
          <TableFooter
            as="tfoot"
            fullWidth={true}
          >
            <TableRow
              as="tr"
              cellAs="td"
            >
              <TableHeaderCell
                as="th"
                colSpan="4"
              >
                Built with:
                <List
                  bulleted={true}
                  horizontal={true}
                >
                  <ListItem
                    as="a"
                    href="https://facebook.github.io/react/"
                  >
                    React
                  </ListItem>
                  <ListItem
                    as="a"
                    href="https://react.semantic-ui.com/"
                  >
                    Semantic UI React
                  </ListItem>
                </List>
              </TableHeaderCell>
            </TableRow>
          </TableFooter>
        </Table>,
        "_debugID": 2,
        "_renderedOutput": <Table
          as="table"
          celled={true}
          fixed={true}
          selectable={true}
          sortable={true}
          striped={true}
          textAlign="center"
        >
          <TableHeader
            as="thead"
          >
            <TableRow
              as="tr"
              cellAs="td"
            >
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                Camper
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                Points in past 30 days
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                All time points
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
              >
                Last Active
              </TableHeaderCell>
            </TableRow>
          </TableHeader>
          <TableBody
            as="tbody"
          />
          <TableFooter
            as="tfoot"
            fullWidth={true}
          >
            <TableRow
              as="tr"
              cellAs="td"
            >
              <TableHeaderCell
                as="th"
                colSpan="4"
              >
                Built with:
                <List
                  bulleted={true}
                  horizontal={true}
                >
                  <ListItem
                    as="a"
                    href="https://facebook.github.io/react/"
                  >
                    React
                  </ListItem>
                  <ListItem
                    as="a"
                    href="https://react.semantic-ui.com/"
                  >
                    Semantic UI React
                  </ListItem>
                </List>
              </TableHeaderCell>
            </TableRow>
          </TableFooter>
        </Table>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Body />,
}
`;
